test/Target/llvmir.mlir

```diff
@@ -825,3 +825,9 @@ func @stringconstant() -> !llvm<"i8*"> {
   llvm.return %1 : !llvm<"i8*">
 }
 
+// CHECK-LABEL: define void @noreach() {
+// CHECK-NEXT:    unreachable
```

Comment:
We don't need to check the trailing '}' here.

Reply:
I see your point, however I left it in there to be consistent with some similar checks earlier in the file.

Reply:
It's alright to diverge here, we can clean the rest in a followup.

Reply:
It looks like these last few tests cherry pick the LLVM IR output. If that's the new preferred pattern, perhaps I should remove the CHECK-LABEL line as well.